pkg/reconciler/chart/helm.go

```diff
@@ -31,20 +32,44 @@ func NewHelmClient(chartDir string, logger *zap.SugaredLogger) (*HelmClient, err
 	}, nil
 }
 
+func (c *HelmClient) getPath(component *Component) (string, error) {
+	if component.isExternalComponent() {
+		path := ""
+		err := filepath.WalkDir(c.chartDir, func(p string, d fs.DirEntry, err error) error {
+			if filepath.Base(p) == "Chart.yaml" && path == "" {
+				path = p
+			}
+			return nil
+		})
+		if err != nil {
+			return "", err
```

Comment:
I don't think this can happen because the `WalkDirFunc` doesn't return an error ever, but linter was complaining...